examples/trigonometry.R

```diff
@@ -0,0 +1,71 @@
+# https://processing.org/tutorials/trig/
```

Comment:
Good -- whenever an example is based on a reference example, briefly document. We will expand these later.

We should base examples on reference examples as often as possible, especially Java and/or p5 examples -- not just because it is faster to adapt, because users often compare pairs of examples together in order to better understand how to translate between modes.